app/code/core/Mage/Admin/Model/Resource/Acl.php

```diff
@@ -131,9 +133,26 @@ public function loadRules(Mage_Admin_Model_Acl $acl, array $rulesArr)
                 } elseif ($rule['permission'] == 'deny') {
                     $acl->deny($role, $resource, $privileges, $assert);
                 }
+            } catch (Zend_Acl_Exception $e) {
+                if (
+                    !in_array($resource, $this->_orphanedResources)
+                    && strpos($e->getMessage(), "Resource '$resource' not found") !== false
+                ) {
```

Comment:
```suggestion
                if (!in_array($resource, $this->_orphanedResources)
                    && strpos($e->getMessage(), "Resource '$resource' not found") !== false) {
```
phpcs

Reply:
How about  the doc on [PSR12](https://www.php-fig.org/psr/psr-12/):
![image](https://github.com/OpenMage/magento-lts/assets/1106470/4cc6a52c-f8f0-4290-853d-c0d579240888)

Search for "multiple lines".

Reply:
[More info](https://en.wikipedia.org/wiki/PHP_Standard_Recommendation) on PSR.
> PSR-12	Extended Coding Style Guide: It extends, expands and replaces PSR-2, the coding style guide and requires adherence to PSR-1, the basic coding standard.[18]

Reply:
I'll stick with one-line, which will satisfies both PSR-2 and PSR-12.